docs/src/content/docs/es/reference/configuration.mdx

```diff
@@ -568,3 +569,13 @@ starlight({
 ```
 
 Consulta la [Referencia de Plugins](/es/reference/plugins/) para obtener detalles sobre cómo crear tus propios plugins.
+
+### `credits`
+
+Habilita la visualización de un enlace “Construido con Starlight” en el pie de página de tu sitio.
```

Comment:
```suggestion
Habilita la visualización de un enlace “Hecho con Starlight” en el pie de página de tu sitio.
```

When the feature was built live on Discord, people suggested the ["Hecho con Starlight"](https://github.com/withastro/starlight/blob/main/packages/starlight/translations/es.json#L31) translation so I guess both the translation and doc should match.

Not sure if one is better than the other?